smriprep/workflows/anatomical.py

```diff
@@ -614,12 +612,12 @@ def _set_threads(in_list, maximum):
         (n4_correct, t1w_merge, [('output_image', 'in_files')]),
         (t1w_merge, t1w_reorient, [('out_file', 'in_file')]),
         # Combine orientation and template transforms
-        (t1w_conform_xfm, concat_affines, [('out_lta', 'in_lta1')]),
-        (t1w_merge, concat_affines, [('transform_outputs', 'in_lta2')]),
-        (concat_affines, lta_to_itk, [('out_file', 'in_lta')]),
+        (t1w_conform_xfm, merge_xfm, [('out_lta', 'in1')]),
+        (t1w_merge, merge_xfm, [('transform_outputs', 'in2')]),
+        (merge_xfm, concat_xfms, [('out', 'in_xfms')]),
         # Output
         (t1w_reorient, outputnode, [('out_file', 't1w_ref')]),
-        (lta_to_itk, outputnode, [('out_itk', 't1w_realign_xfm')]),
+        (concat_xfms, outputnode, [('out_inv', 't1w_realign_xfm')]),
```

Comment:
I have a vague memory that FreeSurfer's LTAs nomenclature is like the BIDS-Derivatives' `mode-points` of transforms (i.e., the _reference_ is the `to-` field and its data is moved to the _target_ --or `from-`). I'll need to check this offline to make sure we are correctly using and naming these outputs.

Reply:
I haven't seen any ApplyTransform based on these new ITK transforms in sMRIPrep, so I'll need to run a full fMRIPrep run with this to make sure we got the direction of the transform correct.

Reply:
Addressed in https://github.com/poldracklab/smriprep/pull/188/commits/8db93dcff385c2aabd15f626e706bcaa8239d7d5. Checked on one session+run of ds031.